vue-cli-2/cypress.config.js

```diff
@@ -0,0 +1,15 @@
+const { defineConfig } = require("cypress");
+const { startDevServer } = require("@cypress/webpack-dev-server");
+const webpackConfig = require("@vue/cli-service/webpack.config");
+
+module.exports = defineConfig({
+  component: {
+    devServer(cypressDevServerConfig) {
```

Comment:
FYI this isn't the correct way to specify the `devServer` config option. Please see the [Changes in 10.0.0](https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#cfd68e3db8474d6faec70e47870fbd19) doc!